ibm/mas_devops/roles/suite_mustgather/defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
 base_output_dir: "{{ lookup('env', 'BASE_OUTPUT_DIR') }}"
+collect_certificates: "{{ lookup('env', 'COLLECT_CERTIFICATES') | default(false, true) }}"
```

Comment:
Can we add a filter to make this a boolean? Just add `| bool` to the end like `default(false, true) | bool }}"`

Reply:
done
changed to:
collect_certificates: "{{ lookup('env', 'COLLECT_CERTIFICATES') | default(false, true) | bool }}"